packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -140,6 +140,7 @@ import {
 	tagCodeArtifacts,
 	normalizeError,
 } from "@fluidframework/telemetry-utils/internal";
+import { gt } from "semver-ts";
```

Comment:
nit: Consider renaming this import to `semverGT` or `semverGr` or `semverGreater` or something with "semver"